package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "> The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.",
+  "main": "index.js",
+  "deploy": "gh-pages -d dist",
+  "scripts": {
+    "test": "jest --coverage",
+    "build": "webpack",
+    "dev": "webpack serve --open"
+  },
```

Comment:
- [ ] When you run the command `npm run build` it supposes to generate `index.html` and `index.js` files inside the `dist` folder it only generates `index.html` file and `main.bundle.js` and also it throws an error like the image below so please fix that.

![image](https://user-images.githubusercontent.com/68796232/163185108-9efdbb45-b788-411c-95b1-ae06e8c1cc27.png) ![image](https://user-images.githubusercontent.com/68796232/163185139-4fba0af5-f400-4e7a-b711-7a0d90a2b3a3.png)

- [ ] When you run `npm run start` command it supposes to run the live demo server but it doesn't do that so please fix that.

[Hint]: Kindly go back again to the exercise before this project and follow the steps carefully to make every thing works perfectly with you at the end your script inside the `package.json` file suppose to be like that
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Reply:
@SaidRasinlic - It would be nice to add the steps to run your project in the readme file. 🙂